packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.scss

```diff
@@ -40,6 +40,7 @@
   .row {
     display: grid;
     grid-template-columns: repeat(4, minmax(0, 1fr));
+    column-gap: 5rem;
```

Comment:
How does this look in smaller sizes?

Reply:
See 0:07 - 0:14 in the video above.

Reply:
Thought the video was a screenshot... My mistake!